examples/dwarfdump.rs

```diff
@@ -365,7 +365,7 @@ struct Flags {
 impl Flags {
     fn section_name(&self, id: gimli::SectionId) -> Option<&'static str> {
         if self.dwo {
-            id.dwo_name()
+            id.dwo_names().get(0).map(|s| *s)
```

Comment:
This will still only load one of the section names.

Rather than allowing multiple names per section, what if leave `.debug_loc.dwo` and `.debug_loclists.dwo` as separate sections, and choose which one to use in `raw_locations_dwo` based on the version?

Reply:
Yeah ok that makes sense.